library/cwm/src/lib/cwm/multi_status_selector.rb

```diff
@@ -436,7 +437,7 @@ def checkbox_input
         if enabled?
           "<a href=\"#{id}#{INPUT_EVENT_ID}\" style=\"#{text_style}\">#{icon}</a>"
         else
-          "<span style\"#{text_style}\">#{icon}</a>"
+          "<span style=\"#{text_style}\">#{icon}</a>"
```

Comment:
Nice catch!

But _we_ (mostly me) also forget to use the right tag to close it. The typical copy&paste problem, sorry.

```suggestion
          "<span style=\"#{text_style}\">#{icon}</span>"
```